build.sbt

```diff
@@ -379,7 +380,10 @@ val settings = Seq(
   assembly / assemblyOption := (assembly / assemblyOption).value.copy(includeScala = false),
   autoAPIMappings := true,
   pomPostProcess := pomPostFunc,
-  sbtPlugin := false
+  sbtPlugin := false,
+  assembly / assemblyShadeRules ++= Seq(
```

Comment:
This wont do anything because we dont use sbt assembly for putting things onto cluster